lib/src/connection_fsm.dart

```diff
@@ -110,6 +110,15 @@ class _PostgreSQLConnectionStateAuthenticating
           _authenticator =
               createAuthenticator(connection!, AuthenticationScheme.MD5);
           continue authMsg;
+        case AuthenticationMessage.KindClearTextPassword:
+          if (connection!.allowClearTextPassword) {
+            _authenticator =
+                createAuthenticator(connection!, AuthenticationScheme.CLEAR);
+            continue authMsg;
+          }
+          else {
+            break;
```

Comment:
Shouldn't this throw a `PostgreSQLException` with a message that explains that the flag is not set?

Reply:
I kept the original format for that message, it executes at the end of the switch:
```
      completer.completeError(PostgreSQLException(
          'Unsupported authentication type ${message.type}, closing connection.'));
```
Shall I open a  new if-else branch to personalize the message?

Reply:
Ah, ok, this will be fine then...

Reply:
Please confirm the message:
```
completer.completeError(PostgreSQLException(
                'type ${message.type} connections disabled. Set AllowClearTextPassword flag on PostgreSQLConnection to enable this feature.'));
```